app/bundles/CoreBundle/Views/Theme/list.html.twig

```diff
@@ -0,0 +1,153 @@
+{#
+/*
+ * @copyright   2014 Mautic Contributors. All rights reserved
+ * @author      Mautic
+ *
+ * @link        http://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+ #}
```

Comment:
Remove, please

Reply:
Any particular reason this needs to be removed? The PHP templates also carry meta-information like this.

Reply:
Please see https://github.com/mautic/mautic/pull/10970 for the reasons